test/infamy/topology.py

```diff
@@ -136,8 +149,5 @@ def get_infixen(self):
     log = Topology(pydot.graph_from_dot_file(sys.argv[2])[0])
     if log.map_to(phy):
         print(json.dumps(log.mapping))
-        print(json.dumps(tuple(log.get_paths("host", "target"))))
-        sys.exit(0)
 
-    print("{}")
```

Comment:
I know I was the one who removed these during our last session, but these lines should go back :smile:

Reply:
reverted :)